includes/admin/customers/customers.php

```diff
@@ -763,7 +769,7 @@ function edd_customers_view( $customer = null ) {
 			<thead>
 				<tr>
 					<th><?php esc_html_e( 'Email',   'easy-digital-downloads' ); ?></th>
-					<th><?php esc_html_e( 'Actions', 'easy-digital-downloads' ); ?></th>
+					<th><?php esc_html_e( 'Date Added', 'easy-digital-downloads' ); ?></th>
```

Comment:
The physical addresses table uses "First Used" for the column header. It feels like the data is a similar type and maybe we should use the same for each?

Reply:
I like this suggestion.  I don't care which label we use, but consistency is nice.

Reply:
I don't mind that they are different as the context is different. One is an object, the other is a property of an object than can have multiple values related to the context (email/physical address).